cmake/external.cmake

```diff
@@ -141,3 +141,34 @@ function(link_x11 target_name)
         target_link_libraries(${target_name} PRIVATE ${X11_Xtst_LIB})
     endif()
 endfunction()
+
+# libvpx
+function(link_vpx target_name)
+    if (TG_OWT_VPX_PACKAGED_BUILD)
```

Comment:
I don't think we really need such variable for each library. This was a mess in tdesktop and was removed. Can you use TG_OWT_PACKAGED_BUILD instead?

Reply:
Also, I think it's worth to have behavior just like in tdesktop: fallback to the bundled library if packaged one is not found

Reply:
An example of how this is done in tdesktop: https://github.com/desktop-app/cmake_helpers/blob/master/external/dbusmenu_qt/CMakeLists.txt#L10-L22